WordPress/Classes/ViewRelated/NUX/SignupGoogleViewController.swift

```diff
@@ -25,80 +25,129 @@ class SignupGoogleViewController: LoginViewController {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        if !hasShownGoogle {
-            showGoogleScreen()
-            hasShownGoogle = true
+        displayGoogleSingleSignOnIfNeeded()
+    }
+
+    private func displayGoogleSingleSignOnIfNeeded() {
+        guard !hasShownGoogle else {
+            return
         }
+
+        displayGoogleSingleSignOn()
+        hasShownGoogle = true
     }
 
-    private func showGoogleScreen() {
+    private func displayGoogleSingleSignOn() {
```

Comment:
Let's keep `showGoogleScreen `. I don't think `displayGoogleSingleSignOn` is strictly accurate, anyway. The library is `Google Sign-In`

Reply:
Sounds good!